src/MarketingPageAcceptanceTests.TestData/Solutions/GenerateCatalogueItem.cs

```diff
@@ -35,67 +29,9 @@ public static class GenerateCatalogueItem
             return catalogueItem;
         }
 
-        private static string GetSuffix(int solIdLength)
-        {
-            var suffix = string.Empty;
-            for (var i = 0; i < 14 - solIdLength; i++)
-            {
-                suffix += GetRandomCharacter();
-            }
-
-            return suffix;
-        }
-
-        private static string GetRandomCharacter()
-        {
-            var faker = new Faker();
-            List<string> randomChars = new();
-            for (var i = 0; i < 10; i++)
-            {
-                randomChars.Add(faker.Random.AlphaNumeric(1));
-            }
-
-            return RandomInformation.GetRandomItem(randomChars);
-        }
-
-        private static string GetTruncatedTimestamp(string prefix)
-        {
-            var timestamp = DateTimeOffset.Now.ToUnixTimeMilliseconds().ToString();
-            timestamp = timestamp[^prefix.Length..];
-
-            return prefix + timestamp;
-        }
-
-        private static string UniqueSolIdCheck(string prefix, string connectionString)
+        private static string RandomSolId(string supplierid)
```

Comment:
Would `RandomSolutionId` be better here? (As it's not much longer in length and more obvious.)